credentialMixin.js

```diff
@@ -4,33 +4,15 @@
 'use strict';
 
 export const credentialMixin = {
-  data() {
-    return {
-      showFieldValues: false
-    };
-  },
-  created() {
-    // Defaults shown fields to 3 if not defined as prop
-    if(this.fieldQuantity === undefined) {
-      this.fieldQuantity = 3;
-    }
-  },
-  computed: {
-    fields() {
-      if(!(this.credential || this.schema)) {
-        return {};
-      }
-      const {credentialSubject} = this.credential;
-      const fields = {};
-      _createFields(fields, credentialSubject, this.schema);
-      return fields;
-    }
-  },
   props: {
     credential: {
       type: Object,
       required: true,
     },
+    didMap: {
```

Comment:
DIDs shouldn't be required, a non-DID identifier should work. Let's call this `issuerMap` instead.